src/elm/clinical.ts

```diff
@@ -84,7 +88,7 @@ export class InValueSet extends Expression {
       return false;
     }
     if (this.valueset == null) {
-      throw new Error('ValueSet must be provided to InValueSet function');
+      throw new Error('ValueSet must be provided to InValueSet expression');
     }
```

Comment:
These two checks on `this.code` and `this.valueset` will never be hit because they should not be possible with valid elm. The constructor will error when it is being built if the elm is bad. These checks can be removed.